etc/schema.json

```diff
@@ -6757,13 +6768,30 @@
                             }
                         },
                         "notafter": {
-                            "$ref": "#/$defs/tls_date"
+                            "$ref": "#/$defs/tls_date",
+                            "suricata": {
```

Comment:
@jasonish these are not picked up

Reply:
Hmm..  I think the $ref here doesn't allow for other fields, so "suricata" should go in the $ref seciton. But I can see how that breaks down when ref's are re-used. Will have to do some object merging.

Reply:
Ok, fixed in #12739